app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    integer = split_integer(8, 2)
+    assert sum(integer) == 8
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    integer = split_integer(32, 6)
+    assert integer == [5, 5, 5, 5, 6, 6]
```

Comment:
Ensure that the implementation of `split_integer` matches the expected output `[5, 5, 5, 5, 6, 6]` for `split_integer(32, 6)`, as this is correct according to the task description.

Reply:
@mateacademy-ai-mentor so what i do?